.github/workflows/fix-typos.yml

```diff
@@ -0,0 +1,19 @@
+name: Automatically fix typos
+on:
+  push:
+    branches:
+      - master
```

Comment:
Why only the master branch?

OctoPrint's development all happens on branches that are not master.

Reply:
<img width="319" alt="Captura de ecrã 2022-08-02, às 22 13 10" src="https://user-images.githubusercontent.com/29093946/182474356-43052dd7-232b-485f-b960-d01ede75503f.png">

in this case I added master because was the default branch

Reply:
Please see [CONTRIBUTING.md](https://github.com/OctoPrint/OctoPrint/blob/master/CONTRIBUTING.md) and the link to [the branches section of the documentation](https://docs.octoprint.org/en/master/development/branches.html) that explains it.

Reply:
thanks for sending! i can update to include those instead 🔨

Reply:
considering your comment [here](https://github.com/OctoPrint/OctoPrint/pull/4603#pullrequestreview-1059405775) probably this action is not the most suitable after all, will remove 🔨